src/server/index.ts

```diff
@@ -68,6 +70,7 @@ initDb()
       app.set('trust proxy', trustProxy)
     }
 
+    app.set('views', path.resolve(__dirname, './views'))
```

Comment:
I am curious to find out before this line of code and the `postbuild` line in `package.json`, how was 404 pages served in previous builds?

Reply:
it previously relied on the default view folder being located in the top level directory where the node process was launched.